0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -167,14 +115,34 @@ Content-Type: application/spsp4+json
 }
 ```
 
-### Payment Setup
+## Model of Operation
+
+### Establishing a connection
+
+We assume that the client knows the server's SPSP endpoint (see [Payment Pointers](../0026-payment-pointers/0026-payment-pointers.md)).
+
+1. The user's SPSP client queries the server's SPSP Endpoint.
+
+2. The SPSP endpoint responds with the server info, namely the server's ILP address (`destination_account`) and the shared secret (`shared_secret`) to be used in STREAM. 
+    * The `destination_account` SHOULD be used as the STREAM destinationAccount.
+    * The `shared_secret` SHOULD be decoded from base64 and used as the STREAM sharedSecret.
+  
+    It MAY respond with additional information if it is an invoice server or a pull payment server. For more information, see [SPSP Invoices](../0035-spsp-invoices/0035-spsp-invoices.md) and [SPSP Pull Payments](../0036-spsp-pull-payments/0036-spsp-pull-payments.md).
+
+3. The SPSP client establishes a STREAM connection to the server using the server's ILP address and shared secret.
+
+### Simple push payment
```

Comment:
I wonder if any of this is necessary, it doesn't seem normative and yet it might make people think it is?

Reply:
I could add a disclaimer but I would leave it in there because there is no other good place to put it and it serves as a minimal example. We discussed that in the [forum](https://forum.interledger.org/t/push-payments-and-invoices/103).

Reply:
Not a deal-breaker for me. 

The thing is this is an example of a STREAM payment and doesn't say much about SPSP this suggests again that this whole spec COULD form part of an appendix to the STREAM spec.